src/prefect/tasks/kubernetes/pod.py

```diff
@@ -571,3 +573,118 @@ def run(
         api_client.replace_namespaced_pod(
             name=pod_name, namespace=namespace, body=body, **kube_kwargs
         )
+
+
+class ReadNamespacedPodLogs(Task):
```

Comment:
Can you add this to the api docs (add to the list of classes [here](https://github.com/PrefectHQ/prefect/blob/928d2b9e9b54847e3d021a544c9cdeb8ea1ba10b/docs/outline.toml#L312)).